packages/node-cli/src/commands/object/update/UpdateInputHandler.js

```diff
@@ -42,6 +49,47 @@ module.exports = class UpdateInputHandler extends BaseInputHandler {
 	}
 
 	async getParameters(params) {
+		const foundXMLFiles = this.searchFilesFromObjectsFolder();
+		let filteredObjects = await this.getObjectsToSelect(foundXMLFiles);
+		const selectedScriptIds = await this.selectObjectsScriptIds(foundXMLFiles, filteredObjects);
```

Comment:
It think we can name these like:
filteredObjects -> filteredObjectsList
selectObjectsScriptsIds(foundXMLFiles, filteredObjects) method -> getSelectedScriptsIds(filteredObjects)
Actually the method should only use the filteredObjects/filteredObjectsList.